client/src/Components/Register.jsx

```diff
@@ -1,7 +1,201 @@
+            value={registerData.lastName}
+            onChange={(e) => handleChange(e)}
+            required
+          />
+        </div>
+        <br className="login-signup-form-break" />
+        <div className="login-signup-email">
+          <FaRegEnvelope className="login-signup-icon" />
+          <input
+            type="email"
+            id="email"
+            aria-label="Email"
+            placeholder="Email"
+            value={registerData.email}
+            onChange={(e) => handleChange(e)}
+            required
+          />
+        </div>
+        <br className="login-signup-form-break" />
+        <div className="login-signup-zipCode">
+          <GrLocation className="login-signup-icon" />
+          <input
+            type="zipCode"
+            id="zipCode"
+            aria-label="Zip Code"
+            placeholder="Zip Code"
+            value={registerData.zipCode}
+            onChange={(e) => handleChange(e)}
+            required
+          />
+        </div>
+        <br className="login-signup-form-break" />
+        <div className="login-signup-username">
+          <FaRegUser className="login-signup-icon" />
+          <input
+            type="text"
+            id="username"
+            aria-label="Username"
+            placeholder="Username"
+            value={registerData.username}
+            onChange={(e) => handleChange(e)}
+            required
+          />
+        </div>
+        <br className="login-signup-form-break" />
+        <div className="login-signup-password">
+          <FiLock className="login-signup-icon" />
+          <input
+            type="password"
+            id="password"
+            aria-label="Password"
+            placeholder="Password"
+            minLength="6"
+            value={registerData.password}
+            onChange={(e) => handleChange(e)}
+            onKeyUp={handlePasswordCheck}
+            required
+          />
+        </div>
+        {(registerData.password.length > 0 && !validPassword)
+          && (
+            <div className="valid-password">
+              <small>Password must be a minimum of 6 characters</small>
+            </div>
+          )}
+        <br className="login-signup-form-break" />
+        <div className="login-signup-confirm-password">
+          <FiLock className="login-signup-icon" />
+          <input
+            type="password"
+            id="confirmPassword"
+            aria-label="Confirm Password"
+            placeholder="Confirm Password"
+            value={registerData.confirmPassword}
+            onChange={(e) => handleChange(e)}
+            onKeyUp={handlePasswordCheck}
+            required
+          />
+        </div>
+        {!passwordMatch
+          && (
+            <div className="matching-password">
+              <small>Passwords do not match</small>
+            </div>
+          )}
+      </form>
+
+      <div className="lower-buttons login-signup-buttons">
+        <button
```

Comment:
food for thought on Form tag here, should this button be within the <form tag for sematicness? imo I do like the button instead of <input type submit to "submit" the form details, but im not sure if best to still be inside the form tag or not, can disregard for another pr too.